src/sources/internal_logs.rs

```diff
@@ -100,8 +115,20 @@ async fn run(
             log.insert(host_key.as_str(), hostname.to_owned());
         }
         log.insert(pid_key.as_str(), pid);
```

Comment:
I spy with my little eye, two fields that aren't namespaced or part of the schema 👁️

Reply:
:+1: As we discussed offline, I'll address that and now I can improve on what to search for to know what needs to be included in the schema. Thanks for pointing it out~

Reply:
addressed in latest changeset